modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/pages/Invites/index.js

```diff
@@ -219,25 +228,30 @@ const Invites = () => {
 							invite={invite}
 							key={index}
 							options={accountRoles}
+							placeholderEmail={`username@${
+								project ? project.code.toLowerCase() : 'example'
+							}.com`}
 						/>
 					))}
 				</ClayForm.Group>
 
-				<BaseButton
-					borderless
-					className="mb-3 ml-3 mt-2 text-brand-primary"
-					onClick={() => {
-						setBaseButtonDisabled(false);
-						setFieldValue('invites', [
-							...values.invites,
-							getInitialInvite(),
-						]);
-					}}
-					prependIcon="plus"
-					small
-				>
-					Add More Members
-				</BaseButton>
+				{values.invites.length < 10 && (
```

Comment:
Why 10? this could be a constant to represent their meaning;

Something like: MAXIMUM_INVITES_COUNT = 10;

Reply:
Sure, I'll be adding a variable to that